src/textual/_segment_tools.py

```diff
@@ -121,9 +121,9 @@ def line_trim(segments: list[Segment], start: bool, end: bool) -> list[Segment]:
     """Optionally remove a cell from the start and / or end of a list of segments.
 
     Args:
-        segments: A line (list of Segments)
-        start: Remove cell from start.
-        end: Remove cell from end.
+        segments (list[Segment]): A line (list of Segments)
```

Comment:
I think these types get pulled in from the signature in the docs.

Reply:
Whoops, reverted too far :)